### PR TITLE
Added FAQ Page

### PR DESCRIPTION
**Pull request by AdityaBhate**
### Description

Added a new FAQ page and updated the Navbar and Footer with a link to the FAQ page.

### Related Issue

#238

### Changes Made

- Created a dedicated FAQ page with an Accordion component to display questions and answers.
- Updated the Navbar and Footer to include a link to the FAQ page.

### Screenshots

image

### Checklist

- [x] I have tagged the issue in this PR.
- [x] I have attached necessary screenshots.
- [x] I have provided a short description of the PR.
- [x] I ran `yarn build` and build is successful
- [x] My code follows the style guidelines of this project.
- [x] I have added necessary documentation (if applicable)

### CODE REVIEW

File: app/faqs/page.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  {
+    question: "Is PearAI an extension or an app?",
+    answer:
+      "PearAI is an app and requires you to download it. Being a VSCode extension severely limits the modifications we can do. In order to provide the best UX possible, we need full control of code changes within the app.",
+  },
+];
+
+const FAQ: React.FC = () => {
+  return (
+    <div className="mx-auto mt-10 flex w-full max-w-screen-lg flex-col gap-4 px-4 py-8 lg:px-8 lg:py-12">
```
Comment (ICEPrey) [inline app/faqs/page.tsx:45]:
```suggestion
     <section className="mx-auto mt-10 flex w-full max-w-screen-lg flex-col gap-4 px-4 py-8 lg:px-8 lg:py-12">
```

File: app/faqs/page.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      <div className="mb-4">
+        <h4
+          className="whitespace-pre-line text-2xl font-medium text-primary-700 lg:text-4xl"
+          data-aos="fade-up"
+        >
+          Frequently Asked Questions
+        </h4>
```
Comment (ICEPrey) [inline app/faqs/page.tsx:52]:
```suggestion
        <h2
          className="whitespace-pre-line text-2xl font-medium text-primary-700 lg:text-4xl"
          data-aos="fade-up"
        >
          Frequently Asked Questions
        </h2>
```

File: app/faqs/page.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          <Link
+            className="underline"
+            rel="noopener noreferrer"
+            target="_blank"
+            href="https://discord.gg/AKy5FmqCkF"
+          >
+            Discord
+          </Link>
+          .
+        </div>
```
Comment (ICEPrey) [inline app/faqs/page.tsx:67]:
```suggestion
        <p
          className="whitespace-pre-line text-base text-gray-600 lg:text-base"
          data-aos="fade-up"
        >
          Can&apos;t find the answer you&apos;re looking for? Join our{" "}
          <Link
            className="underline"
            rel="noopener noreferrer"
            target="_blank"
            href="https://discord.gg/AKy5FmqCkF"
          >
            Discord
          </Link>
          .
        </p>
```

File: app/faqs/page.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      </div>
+      <Accordion type="single" collapsible>
+        {faqData.map((item, index) => (
+          <AccordionItem key={index} value={`{${index + 1}}`}>
```
Comment (ICEPrey) [inline app/faqs/page.tsx:71]:
```suggestion
          <AccordionItem key={index} value={String(index + 1)}>
```
explicitly convert string

File: app/faqs/page.tsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+              className="underline"
+              target="_blank"
+              href="https://docs.google.com/presentation/d/1zR9-7DTlb2PcsnapryZw8jHSkLTs9JxeXth4nyeemAQ/edit?usp=sharing"
+            >
+              Contributing 101
+            </Link>
+          </AccordionContent>
+        </AccordionItem>
+      </Accordion>
+    </div>
```
Comment (ICEPrey) [inline app/faqs/page.tsx:97]:
```suggestion
    </section>
```

**Review by ICEPrey — CHANGES_REQUESTED** [review]
Minor picky changes for semantic html, for SEO

File: app/faqs/page.tsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import Link from "next/link";
+
+interface FAQItem {
+  question: string;
+  answer: string;
+}
+
+const faqData: FAQItem[] = [
```
Comment (Fryingpannn) [inline app/faqs/page.tsx:16]: Let's put this faqData into a separate file

File: components/ui/navLink.tsx [hunk 7]
```diff
@@ -60,6 +60,12 @@ const Navigation: React.FC = () => {
       isExternal: false,
       mobile: false,
     },
+    {
+      label: "FAQs",
+      path: "/faqs",
```
Comment (Fryingpannn) [inline components/ui/navLink.tsx:65]: remove `s`, should just be faq (both path and label)

**Review by Fryingpannn — DISMISSED** [review]

**Pull request merged**